frontend/src/containers/DocList.js

```diff
@@ -166,37 +193,46 @@ class DocList extends Component {
  * @prop {object} windowType
  */
 DocList.propTypes = {
-  breadcrumb: PropTypes.array.isRequired,
-  dispatch: PropTypes.func.isRequired,
-  includedView: PropTypes.object.isRequired,
-  indicator: PropTypes.string.isRequired,
+  includedView: PropTypes.object,
   latestNewDocument: PropTypes.any,
   modal: PropTypes.object.isRequired,
   overlay: PropTypes.object,
-  pathname: PropTypes.string.isRequired,
-  pluginModal: PropTypes.object,
   processStatus: PropTypes.string.isRequired,
   query: PropTypes.object.isRequired,
+  pathname: PropTypes.string.isRequired,
   rawModal: PropTypes.object.isRequired,
-  windowType: PropTypes.any,
+  windowType: PropTypes.string,
```

Comment:
Like I said above. We can expect in here windowId and pass it along from the routes.js
 file as windowId instead of windowType

Reply:
agree. renamed it if u touch it.